packages/gatsby/src/redux/index.js

```diff
@@ -73,6 +74,16 @@ const store = Redux.createStore(
 
 // Persist state.
 function saveState() {
+  if (process.env.DANGEROUSLY_DISABLE_OOM) {
+    // meausure how many people are using DANGEROUSLY_DISABLE_OOM in build statements
+    telemetry.decorateEvent(`BUILD_END`, {
```

Comment:
@jamo wdyt of this? Maybe I should just omit it 😛

Reply:
I like this a lot! 
We(I) need to allow this key in the API before we can ship it.

Reply:
Maybe it's better to omit this for now? So the PR isn't blocked by this?
cc @jamo

Reply:
sorry for the delays and a good call.